doc/mkdocs/docs/api/basic_json/erase.md

```diff
@@ -56,7 +70,8 @@ void erase(const size_type idx);
    is returned.
 3. Number of elements removed. If `ObjectType` is the default `std::map` type, the return value will always be `0`
    (`key` was not found) or `1` (`key` was found).
-4. /
+4. See 3.
+5. /
```

Comment:
Especially seeing it out of context I never quite know what the slash is supposed to mean.

It's used all over and could be clearer.

Reply:
I could replace it with "(none)" just like cppreference is doing.

Reply:
```
api/basic_json/basic_json.md:1. /
api/basic_json/basic_json.md:3. /
api/basic_json/basic_json.md:4. /
api/basic_json/basic_json.md:6. /
api/basic_json/basic_json.md:8. /
api/basic_json/insert.md:5. /
api/basic_json/to_bson.md:2. /
api/basic_json/to_cbor.md:2. /
api/basic_json/to_msgpack.md:2. /
api/basic_json/to_ubjson.md:2. /
api/basic_json/erase.md:5. /
api/basic_json/erase.md:2. /
api/basic_json/erase.md:5. /
```

Would you like me to replace all occurrences?

Reply:
Yes, please.